test/runtests.jl

```diff
@@ -17,7 +20,7 @@ mutable struct LightDark1DUpperBound
 end
 
 function lower_bound(lb::LightDark1DLowerBound, p::LightDark1D, s::LightDark1DState)
-    _, _, r = generate_sor(p, s, init_lower_action(p), lb.rng)
+    _, _, r = gen(DDNOut{(:sp,:o,:r)}(), p, s, init_lower_action(p), lb.rng)
```

Comment:
Should this be just `DDNOut(:r)`?

Reply:
Yes, when tested agains POMDPs v0.8 it should be, but I wanted this to pass against POMDPs v0.7.3, and it only knows about `generate_sor`